src/main/java/org/hypertrace/gradle/code/style/CodeStylePlugin.java

```diff
@@ -34,22 +34,21 @@ private void configureFormatting(SpotlessExtension spotlessExtension) {
         });
 
     spotlessExtension.kotlinGradle(
-        format -> format
-            .ktlint("0.41.0")
-            .userData(
-                new HashMap<String, String>() {
-                  {
-                    put("indent_size", "2");
-                    put("continuation_indent_size", "2");
```

Comment:
This was deprecated and ignored long ago, no intended change by removing it - https://github.com/pinterest/ktlint/releases/tag/0.22.0